workers/unity/Packages/com.improbable.gdk.core/Systems/CustomSpatialOSSendSystem.cs

```diff
@@ -3,6 +3,13 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Base class for custom replication.
```

Comment:
Replication isn't the word we are using. That is the unreal term :P

Reply:
We use that word quite a bit in the code base already

Reply:
Also what word do we canonical use?